src/apps/content-editor/src/app/components/ReorderNav/ReorderNav.js

```diff
@@ -72,7 +72,7 @@ class ReorderNav extends Component {
       });
     }
     this.setState({
-      [name]: [...this.props.nav.find(item => item.ZUID === name).children],
+      [name]: [...this?.props?.nav.find(item => item.ZUID === name).children],
```

Comment:
I do not think this will solve the  core issue from Sentry, `e.props.nav.find(...) is undefined`. As the message indicates the result from the `find` function is what is undefined. Which ultimately makes the `.children` reference in valid.

Reply:
I fixed the Draglist issue and hopefully that may resolve the `children` bug.